### PR TITLE
CRNS-177: v2 directory structure

### PR DESCRIPTION
**Pull request by virdesai**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: expo-package/v2.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+if (manifest.sdkVersion.split('.')[0] >= 33) {
+  ImagePicker = require('expo-image-picker');
+  Permissions = require('expo-permissions');
+  DocumentPicker = require('expo-document-picker');
+} else {
+  ImagePicker = Expo.ImagePicker;
+  Permissions = Expo.Permissions;
+  DocumentPicker = Expo.DocumentPicker;
```
Comment (vishalnarkhede) [inline expo-package/v2.js:27]: we can remove this I guess ... now that we don't support expo < 39

File: src/components/Attachment/__tests__/Attachment.test.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ import {
   generateGiphyAttachment,
   generateImageAttachment,
   generateImgurAttachment,
-} from 'mock-builders/generator/attachment';
+} from '../../../mock-builders/generator/attachment';
```
Comment (vishalnarkhede) [inline src/components/Attachment/__tests__/Attachment.test.js:18]: Any reason why are we doing this?

**Review by vishalnarkhede — APPROVED** [review]
LGTM. Just couple of comments!!

File: src/components/Attachment/__tests__/Attachment.test.js (hunk 2 above)
Comment (virdesai) [reply src/components/Attachment/__tests__/Attachment.test.js:18]: for consistency with the way we import local files everywhere else

File: expo-package/v2.js (hunk 1 above)
Comment (virdesai) [reply expo-package/v2.js:27]: sure?

**Pull request merged**